.github/workflows/benchmark.yml

```
@@ -58,11 +58,8 @@ jobs:
       # Fail the job based on the benchmark output
       - name: Finalize job
         run: |
-          REPORT="${{ steps.benchmark.outputs.REPORT }}"
-          IS_SLOWER=$(grep "At least one benchmark is slower than the main branch" <<< "$REPORT")
-          IS_MISSING=$(grep "Missing benchmarks" <<< "$REPORT")
-
-          if [[ $IS_SLOWER || $IS_MISSING ]]; then
+          report="${{ steps.benchmark.outputs.REPORT }}"
+          if [[ "$report" == *"At least one benchmark is slower than the main branch"* || "$report" == *"Missing benchmarks"*]]; then
```

Sorry for the red CI. 😞 

I think here you should try:

```suggestion
          if [[ ("$report" == *"At least one benchmark is slower than the main branch"*) || ("$report" == *"Missing benchmarks"*) ]]; then
```

These are the two command result that is tested in the if condition.

---

Your solution was correct, but the blank spaces were unicode characters instead of regular spaces. Thanks for the help!